threedi_modelchecker/migrations/versions/0200_initial.py

```diff
@@ -37,13 +37,43 @@ def create_table_if_not_exists(table_name, *args, **kwargs):
         return op.create_table(table_name, *args, **kwargs)
 
 
+def upgrade_173():
+    """This implements the latest migration from the old stack:
+
+    0172_auto__del_v2initialwaterlevel__del_field_v2orifice_max_capacity__del_f
+    """
+    if "v2_initial_waterlevel" not in existing_tables:
+        return  # skip this migration (probably already done)
+
+    op.drop_table("v2_initial_waterlevel")
+
+    with op.batch_alter_table("v2_orifice") as batch_op:
+        batch_op.drop_column("max_capacity")
+
+    with op.batch_alter_table("v2_impervious_surface") as batch_op:
+        batch_op.drop_column("function")
+
+    with op.batch_alter_table("v2_pipe") as batch_op:
+        batch_op.drop_column("pipe_quality")
+
+    with op.batch_alter_table("v2_culvert") as batch_op:
```

Comment:
Note: I don't think I have to drop/recreate the v2_culvert_view as is done in the original. The only change is that the columns become nullable, which does not break the view.